cmd/root.go

```diff
@@ -115,3 +128,34 @@ func showKernelVersion() {
 		log.Infof("Kernel version: %s", strings.TrimSpace(string(output)))
 	}
 }
+
+func onLimitReached() bool {
+	shouldExit := false
+	if !captureEnded {
+		if strings.Contains(options, "background=true") {
+			captureEnded = true
+			resetTerminal()
+			out, err := exec.Command("/oc-netobserv", "stop").Output()
+			if err != nil {
+				log.Fatal(err)
+			}
+			fmt.Printf("%s", out)
+			fmt.Print(`Thank you for using...`)
+			printBanner()
+			fmt.Print(`
+
+	- Download the generated output using 'oc netobserv copy' command
```

Comment:
not sure why, I've seen sometimes this message displayed, and sometimes not

Reply:
That's strange. The `captureEnded` flag has been introduced on purpose to avoid `updateTable` to clear the display :thinking: 

When it's not showing, what do you see ?

Reply:
that one should do the trick
https://github.com/netobserv/network-observability-cli/pull/102/commits/74cb956767f045108deb62448eb7630c07fc2b72